### DISCUSSION TITLE
Welcome to Dip Discussions!

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/Dip#136] [General]
## 👋 Welcome!
  As you may have heard, Github has been developing a new feature called Discussions, which is now being rolled 
  out in beta to all projects. We envision Discussions as a way of moving the kinds of conversations that currently 
  take place on the old Mailman list serve directly to where development is taking place! To take this thing for a test 
  drive, please post here when you would otherwise post to the list serve. This is a place to connect with other members 
  of our community on topics not suited for opening an Issue. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mosu001** [comment]
Kia ora (hi) @tkralphs Ted, I started Dippy many years (as you know :) )to provide a easy-to-use PuLP interface to the excellent functionality of Dip. Still working on making changes/additions as I continue to use Dippy/Dip for teaching, research and research consulting. Noho ora mai (stay safe), Mike
